Examples/src/components/Examples/Charts2D/BasicChartTypes/HeatmapChart/angulartest.ts

```diff
@@ -0,0 +1,137 @@
+import { Component, OnInit, OnDestroy, ViewChild, ElementRef } from '@angular/core';
```

Comment:
please replace the 
`Examples/src/components/Examples/Charts2D/BasicChartTypes/HeatmapChart/angular.ts`
with this file